cg-spec/editors_draft.html

```diff
@@ -703,7 +707,7 @@ <h3>Discussion</h3>
 
       The resulting RDF-star dataset consists of
       an empty default graph, and a graph named `http://example.org/#G`
-      with two RDF-star triples:</p>
+      with two RDF-star triples:h
```

Comment:
```suggestion
      with two RDF-star triples:</p>
```
or whatever was meant to be here, which I do know wasn't `h`.